user_tools/src/spark_rapids_tools/storagelib/adls/adlspath.py

```diff
@@ -14,9 +14,24 @@
 
 """Wrapper implementation for ADLS remote path"""
 
+import os
+
 from ..csppath import CspPath, register_path_class
 
 
 @register_path_class("adls")
 class AdlsPath(CspPath):
+    """Implementation for ADLS paths"""
+
     protocol_prefix: str = "abfss://"
+
+    @classmethod
+    def get_abfs_account_name(cls, path: str) -> str:
```

Comment:
nit would be nice to have comment with  format.
abfss://<file_system>@<account_name>.dfs.core.windows.net/<path>/<file_name>

Reply:
thanks @tgravescs! updated this.